src/commonMain/kotlin/exchange.dydx.abacus/calculator/MarginCalculator.kt

```diff
@@ -237,18 +251,29 @@ internal object MarginCalculator {
         return if (isCorrectSign) transferAmount else null
     }
 
-    private fun getIsIncreasingPositionSize(
-        parser: ParserProtocol,
-        subaccount: Map<String, Any>?,
-        tradeInput: Map<String, Any>?,
-    ): Boolean {
-        return getPositionSizeDifference(parser, subaccount, tradeInput)?.let { it > 0 } ?: true
+    /**
+     * @description Calculate the amount of collateral to transfer into child subaccount for an isolated margin trade.
+     */
+    fun getIsolatedMarginTransferInAmountForTradeTyped(
```

Comment:
nit: `Typed` -> `Type`

Reply:
mmmm i mean that the inputs are typed, that's why i added `Typed` at the end -- is there a better naming that could suggest that?

Reply:
or i can just have the same name which is kinda ok since the caller can infer based on inputs